conf/gulp-tasks/bundle/bundle.js

```diff
@@ -11,6 +11,7 @@ const {
   TRANSLATION_FLAGGER_REGEX,
   SPEECH_RECOGNITION_LOCALES_SUPPORTED_BY_EDGE
 } = require('../../i18n/constants');
+const rollupJson = require('@rollup/plugin-json');
```

Comment:
For my own edification, was this necessary for the PR or was this a separate attempt at optimizing our bundle size?

Reply:
it was necessary because answers-core now imports its own package.json in the src
I'm not entirely happy that if somebody wants to use rollup with core they need to include an extra plugin
if you want we can revisit the answers-core implementation

Reply:
![image](https://user-images.githubusercontent.com/23005393/158656694-65faace0-967e-488d-a22b-c7029c120491.png)